http-common/src/connector.rs

```diff
@@ -49,21 +49,17 @@ impl Incoming {
             + 'static,
         <H as hyper::service::Service<hyper::Request<hyper::Body>>>::Future: Send,
     {
-        const READ_WRITE_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(5);
         const MAX_REQUESTS_PER_USER: usize = 10;
 
         match self {
             Incoming::Tcp { listener } => loop {
                 let (tcp_stream, _) = listener.accept().await?;
-                let mut timeout_stream = tokio_io_timeout::TimeoutStream::new(tcp_stream);
-                timeout_stream.set_read_timeout(Some(READ_WRITE_TIMEOUT));
-                timeout_stream.set_write_timeout(Some(READ_WRITE_TIMEOUT));
 
                 // TCP is available in test builds only (not production). Assume current user is root.
                 let server = crate::uid::UidService::new(0, server.clone());
                 tokio::spawn(async move {
                     if let Err(http_err) = hyper::server::conn::Http::new()
-                        .serve_connection(Box::pin(timeout_stream), server)
+                        .serve_connection(Box::pin(tcp_stream), server)
```

Comment:
Don't need `Box::pin()`: https://github.com/Azure/iot-identity-service/commit/fcaade3b329994e2818a958f7f83c3216c5e075f#diff-8bd6e0799ebba2043c7b5d189504fdd1d5166ccc415dd9ef6675f4a0b3aaf76fL55

Same for `unix_stream` below: https://github.com/Azure/iot-identity-service/commit/fcaade3b329994e2818a958f7f83c3216c5e075f#diff-8bd6e0799ebba2043c7b5d189504fdd1d5166ccc415dd9ef6675f4a0b3aaf76fL70

Reply:
Fixed